project/src/index.tsx

```diff
@@ -12,9 +14,11 @@ const root = ReactDOM.createRoot(
 
 root.render(
   <React.StrictMode>
-    <App
-      offers = {offers}
-      reviews = {reviews}
-    />
+    <Provider store={store}>
+      <App
+        offers = {offers}
```

Comment:
вокруг равно не нужны пробелы
```suggestion
        offers={offers}
```